tests/modules/shared/utilities/test_usage_stats.py

```diff
@@ -13,9 +13,24 @@
 
 def test_that_convert_failures_to_failure_count_returns_correct_count():
     list_of_failure = [
```

Comment:
Question, why are there no new unit tests in `secureli/modules/pii_scanner/pii_scanner.py`?

Reply:
There are tests here:
tests/modules/shared/utilities/test_usage_stats.py lines 16-33
tests/actions/test_action.py lines 142 - 145